recipes/freetype/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.5.5" %}
+
+package:
+    name: freetype
+    version: {{ version }}
+
+source:
+    fn: freetype-{{ version }}.tar.bz2
+    url: http://download.savannah.gnu.org/releases/freetype/freetype-{{ version }}.tar.bz2
+    sha256: 387bcc4b780b12484aa2ec9f7db1a55d8286eb5639f45fbc0fbba7a4e5a1afb9
+    patches:
+        - subpixel.patch
```

Comment:
> Then please don't add this package to conda-forge and just the default channel one.

I'm not sure that skip a potential dicey (legally) patch is reason enough to avoid building this. Besides with improvements, to channel selection made by @mcg1969 we should be able to install the `defaults` freetype if needed.

> Not adding this patch will ruin the experience for GUI apps, which I think it's more important for the community than some possible lawsuit.

Hmm...not sure I agree. A lawsuit could mean the end of the community and have potentially bad effects for people who have worked hard to build this. Not reading small fonts will make some users unhappy. They don't really seem equal in magnitude to me.

> By the way, Ubuntu also uses this patch.

Could you provide some information as to how they justify using this patch?